packages/manager/apps/pci-kubernetes/src/pages/detail/log/constants.ts

```diff
@@ -1,10 +1,14 @@
+import { OvhSubsidiary } from '@ovh-ux/manager-react-components';
+
 export const TRACKING_PREFIX =
   'PublicCloud::projects::managed_kubernetes_cluster';
 export const TRACKING_SUFFIX = '_managed_kubernetes_cluster::kubernetes';
 
-export const LOGS_INFO: Record<string, string> = {
+export const LOGS_INFO: Partial<Record<OvhSubsidiary, string>> = {
```

Comment:
Type could be declared just once: 

```suggestion
type TFoo = Partial<Record<OvhSubsidiary, string>>
export const LOGS_INFO: TFoo = {
...
}
```

Reply:
You're right. Done !